### ISSUE TITLE
Error:  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead

### ISSUE THREAD

**Issue by jamezm79** [alievk/avatarify#113]
Is this a known issue?

**Comment by E3V3A** [comment]
@jamezm79 
AFAIK, it's not really an issue, but I agree with you, it should be explained and documented somewhere. I don't even know what package that code belong to.

**Comment by alievk** [comment]
It's a Pytorch warning, you can just ignore for a while.

**Comment by E3V3A** [comment]
@alievk  Yeah, I know, but it's annoying. So in those cases I usually make a new issue at remote repo and link back. Because you never know if it's something they'll ever fix.
